docs/guides/run.md

```diff
@@ -159,3 +165,9 @@ mycontainer3                                        122110  cat              0
 mycontainer3                                        122110  cat              0        0        3         /lib/libc.so.6
 mycontainer3                                        122110  cat              0        0        3         /dev/null
 ```
+
+By default, it will use the `ghcr.io` as the registry, `inspektor-gadget/gadget` as the repository prefix and `latest` as the tag. You can also run them in fully qualified form as:
+
```

Comment:
Do we really need to duplicate it? Can't we have a common section with this information for both ig and kubectl-gadget?

Reply:
Good point. I changed it now to only mention the default values.